code/modules/mob/living/simple_animal/hostile/mining_mobs/curse_blob.dm

```diff
@@ -0,0 +1,124 @@
+/mob/living/simple_animal/hostile/asteroid/curseblob
+	name = "curse mass"
+	desc = "A mass of purple... smoke?"
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	icon_state = "curseblob"
+	icon_living = "curseblob"
+	icon_aggro = "curseblob"
+	move_to_delay = 5
+	vision_range = 20
+	aggro_vision_range = 20
+	idle_vision_range = 20
+	maxHealth = 40 //easy to kill, but oh, will you be seeing a lot of them.
+	health = 40
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	melee_damage_type = BURN
+	attacktext = "slashes"
+	attack_sound = 'sound/effects/curseattack.ogg'
+	throw_message = "passes through the smokey body of"
+	obj_damage = 0
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	sentience_type = SENTIENCE_BOSS
+	layer = LARGE_MOB_LAYER
+	var/doing_move_loop = FALSE
+	var/mob/living/set_target
+	var/timerid
+
+/mob/living/simple_animal/hostile/asteroid/curseblob/Initialize(mapload)
+	. = ..()
+	timerid = QDEL_IN(src, 600)
+	playsound(src, 'sound/effects/curse1.ogg', 100, 1, -1)
+
+/mob/living/simple_animal/hostile/asteroid/curseblob/Destroy()
+	new /obj/effect/temp_visual/dir_setting/curse/blob(loc, dir)
+	doing_move_loop = FALSE
+	deltimer(timerid)
```

Comment:
It actually won't and I remember there being a bug or something due to it... though that might have been pooling so I'll remove it and hope nothing breaks